src/appkit.cs

```diff
@@ -21612,6 +22133,19 @@ public interface NSAccessibility
 		[Abstract]
 		[Export ("isAccessibilitySelectorAllowed:")]
 		bool IsAccessibilitySelectorAllowed (Selector selector);
+
+		[Mac (10, 12)]
+		[Abstract]
+		[Export ("accessibilityRequired")]
+		bool AccessibilityRequired { [Bind ("isAccessibilityRequired")] get; set; }
+	}
+
+	[Protocol]
+	interface NSCollectionViewSectionHeaderView : NSCollectionViewElement
+	{
+		[Mac (10, 12)]
+		[NullAllowed, Export ("sectionCollapseButton", ArgumentSemantic.Assign)]
+		NSButton SectionCollapseButton { get; set; }
```

Comment:
Since this is not an `abstract` (`@required`) property just ensure that the generator does the right thing and generates the Get/Set methods, I do not remember if we do this automagically🎱 now